test/test_jit.py

```diff
@@ -1258,15 +1258,15 @@ class TestModule(torch.nn.Module):
             def __init__(self):
                 super(TestModule, self).__init__()
                 self.conv = torch.nn.Conv2d(1, 20, 5, 1)
-                self.bn = torch.nn.BatchNorm2d(num_features=20)
+                self.bn = torch.nn.BatchNorm2d(num_features=20, eps=1e-1)
 
             def forward(self, x):
                 x = self.conv(x)
                 x = self.bn(x)
                 return x
 
         eager = TestModule()
-        scripted = torch.jit.script(eager)
+        scripted = torch.jit.script(copy.deepcopy(eager))
```

Comment:
why do we need to copy the eager module here?

Reply:
Otherwise the scripted and eager model share the same tensors. It was a bug previously and I think it worked by luck.